### ISSUE TITLE
Don't format sourcery inline code

### ISSUE THREAD

**Issue by Cyberbeni** [nicklockwood/SwiftFormat#833]
I use sourcery to automatically generate public initializer for structs inside framework, the code looks like this before using swiftformat:
```
struct A {
   var a: String

   // sourcery:inline:A.init
   public init(a: String) {
      self.a = a
   }
   // sourcery:end
}
```
And swiftformat inserts a blank line before `// sourcery:end`

I'd like to skip formatting between `// sourcery:inline:` and `// sourcery:end` to avoid changing modified date of the file, so that the compiler doesn't want to check what changed

**Comment by nicklockwood** [comment]
@Cyberbeni if you just disable the `blankLinesBetweenScopes` rule it would probably solve this, but I could consider adding a special case for sourcery comments.

**Comment by Cyberbeni** [comment]
Thanks, I'll disable that until it gets fixed.

**Comment by Cyberbeni** [comment]
The indentation of the whole block should still be fixed by SwiftFormat as Sourcery also aligns to the left side of the comments.

**Comment by RomanTysiachnik** [comment]
You can
- Update your template's indentation to sync it with formatting that `swiftformat` proposes for you
- Or you can execute `swiftformat .` right after you execute `Sourcery/run`. It will instantly update the generated code to the expected one.

**Comment by Cyberbeni** [comment]
If sourcery changes the code and then swiftformat changes it back, that will still update the modification time of the file, making Xcode rebuild it and everything else that depends on it.

I already have it set up, so it doesn't do that for me but it could be useful for others to avoid unexpected build time increase because of using both tools.

**Comment by nicklockwood** [comment]
@Cyberbeni fixed in 0.47.10
